apps/api/src/app/workflows-v2/usecases/generate-preview/construct-payload-from-placeholders-with-defaults.usecase.ts

```diff
@@ -0,0 +1,157 @@
+/* eslint-disable no-param-reassign */
```

Comment:
why do we need this?

Reply:
there is a lint not allowing reassignment, but it's used in this logic